ivy_tests/test_ivy/test_frontends/test_torch/test_linalg.py

```diff
@@ -1035,3 +1040,41 @@ def test_torch_matmul(
         rtol=1e-03,
         atol=1e-06,
     )
+
+
+@handle_frontend_test(
+    fn_tree="torch.linalg.norm",
+    dtype_values_axis=helpers.dtype_values_axis(
+        available_dtypes=helpers.get_dtypes("float"),
+        valid_axis=True,
+        min_value=-1e04,
+        max_value=1e04,
+    ),
+    kd=st.booleans(),
+    ord=helpers.ints(min_value=1, max_value=2),
+    test_with_out=st.just(False),
```

Comment:
Try doing something similar to the ivy.vector_norm test: https://github.com/unifyai/ivy/blob/master/ivy_tests/test_ivy/test_functional/test_core/test_linalg.py#L919
Otherwise perhaps try making a compositional helper which changes the ord and dim arguments based on the shape of the created array